subjects/numofdigits/README.md

```diff
@@ -0,0 +1,43 @@
+## number of digits
+
+### Instructions
+
+Write a function that returns the number of digits in a nonnegative integer n.
```

Comment:
Why not use just "positive integer" instead?

Reply:
Done